### PR TITLE
Feature - global est and project status filter

### PR DESCRIPTION
**Pull request by joefitter**
* Added filter options to get statuses for ui
* Filter by status

### CODE REVIEW

File: lib/routers/search/establishment.js [hunk 1]
```diff
@@ -8,20 +9,31 @@ module.exports = () => {
...
   const searchAndFilter = (Establishment, {
     search,
     limit,
     offset,
-    sort = {}
+    sort = {},
+    status = []
   }) => {
     let query = Establishment.query().eager('asru');
+    status = compact(status);
```
Comment (lennym) [inline lib/routers/search/establishment.js]: `.filter(Boolean)` as a service.

Comment (joefitter) [reply lib/routers/search/establishment.js]: heh

Comment (lennym) [reply lib/routers/search/establishment.js]: Oh, I didn't have beef with this, I just had no idea what it did so went and looked it up in the docs.

Comment (joefitter) [reply lib/routers/search/establishment.js]: heh I replaced it - I grabbed it from elsewhere but you're quite right, silly to use a lib for something so simple

**Pull request merged**